packages/ignite-cli/src/commands/plugin.js

```diff
@@ -20,8 +20,14 @@ const walkthrough = async (context) => {
   // Okay, we'll ask one by one, fine
   return await context.prompt.ask([
     {
+      name: 'boilerplate',
+      message: 'Is this an app boilerplate plugin?',
```

Comment:
Don't hate me, but I think this will be more confusing than helpful :/

I think we can include this, but skip the question and move to a documented flag?